pkg/policy/validate.go

```diff
@@ -236,10 +238,13 @@ func Validate(policy kyverno.PolicyInterface, client *dclient.Client, mock bool,
 		var podOnlyMap = make(map[string]bool) //Validate that Kind is only Pod
 		podOnlyMap["Pod"] = true
 		if reflect.DeepEqual(common.GetKindsFromRule(rule), podOnlyMap) && podControllerAutoGenExclusion(policy) {
-			log.Log.V(4).Info("Pod controllers excluded from autogen require adding of preconditions to also exclude the desired controller(s).")
+			msg := "Policies that match a pod apply to all pods including pods managed by controllers excluded " +
```

Comment:
Suggested:  "Policies that match Pods apply to all Pods including those managed by controllers excluded from autogen. Use preconditions to exclude Pods managed by the controllers excluded in the annotation."